src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -177,41 +175,6 @@ export class SavedObjectsManagementPlugin
     const columnSetup = this.columnService.setup();
     const namespaceSetup = this.namespaceService.setup();
 
-    if (home) {
-      home.featureCatalogue.register({
-        id: 'saved_objects',
-        title: i18n.translate('savedObjectsManagement.objects.savedObjectsTitle', {
-          defaultMessage: 'Saved Objects',
-        }),
-        description: i18n.translate('savedObjectsManagement.objects.savedObjectsDescription', {
-          defaultMessage:
-            'Import, export, and manage your saved searches, visualizations, and dashboards.',
-        }),
-        icon: 'savedObjectsApp',
-        path: '/app/management/opensearch-dashboards/objects',
-        showOnHomePage: false,
-        category: FeatureCatalogueCategory.ADMIN,
-      });
-    }
```

Comment:
same here. i think we should not remove this

Reply:
Updated. Do you mean we should update path like this?

Reply:
should this be `app/objects` ?

Reply:
@raintygao sorry for misunderstanding here. it is `/app/objects`.

Reply:
Never mind. It's my problem.